spec/Candid.md

```diff
@@ -64,7 +64,7 @@ The purpose of an IDL is defining the signature, and thereby the *type* of an ac
 This is a summary of the grammar proposed:
 ```
 <prog>  ::= <def>;* <actor>?
-<def>   ::= type <id> = <datatype> | import <text>
+<def>   ::= type <id> = <datatype> | import <text> | import* <text>
```

Comment:
How about `import service` instead of `import*`?

Reply:
`import_service` then

Reply:
Why not just use the keyword combo, to avoid introducing a new keyword? Prettier too

Reply:
Make sense. Updated.